addons/l10n_gb/data/account_chart_template_data.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
     <data noupdate="1">
         <function model="account.chart.template" name="try_loading">
-            <value eval="[ref('l10n_uk.l10n_uk')]"/>
+            <value eval="[ref('l10n_gb.l10n_gb')]"/>
```

Comment:
Something more explicit for the chart template xml id would be nice, like `l10n_gb.l10n_gb_chart_template`. It seems like the right time to do it.